### PR TITLE
[#52065] 13.2 Documentation: Enable manual project folders for OneDri…

### PR DESCRIPTION
**Pull request by MayaBerd**
…ve/SharePoint storages  https://community.openproject.org/work_packages/52065

[#52065] 13.2 Documentation: Enable manual project folders for OneDrive/SharePoint storages

https://community.openproject.org/work_packages/52065

### CODE REVIEW

File: docs/user-guide/projects/project-settings/file-storages/README.md [hunk 1]
```diff
@@ -41,9 +41,22 @@ In the page that follows, you will be able to select one of the file storages se
...
 
-If you have selected the OneDrive/SharePoint option in the previous step, your OneDrive/SharePoint storage will be added to the project and you can start using it right away.
+If you have selected the OneDrive/SharePoint option in the previous step of storage selection, you will now see the **Project folder** options.
 
-Please refer to the [OneDrive/SharePoint user guide](../../../file-management/one-drive-integration) for further instructions.
+Here you can choose which kind of OneDrive/SharePoint project folder will be the default folder for file uploads for this project when using SharePoint integration.
+
+![Select OndeDrive/SharePoint folder type in OpenProject](onedrive-storage-add-folders.png)
+
+1. **No specific folder**: By default, each user will start at their own home folder when they upload a file.
```
Comment (Kharonus) [inline docs/user-guide/projects/project-settings/file-storages/README.md]: in OneDrive/SharePoint there is no such thing as "user home folder", like in nextcloud. If `no specific folder` is selected, the user will start in the drive root when uploading files or selecting files for linking.

File: docs/user-guide/projects/project-settings/file-storages/README.md [hunk 2]
```diff
@@ -41,9 +41,22 @@ In the page that follows, you will be able to select one of the file storages se
...
+2. **Existing folder with manually managed permissions**: Once you selected this option, you can designate an existing folder as the project folder for this project using the button **Select folder**. The permissions are however not automatically managed. The administrator needs to manually ensure relevant users have access. The selected folder can be used by multiple projects.
```
Comment (Kharonus) [inline docs/user-guide/projects/project-settings/file-storages/README.md]: "The permissions are however not automatically managed**, instead the folder and its content inherits the permissions as configured in the SharePoint site or drive**.

I'd add this, as Microsoft cloud uses a quite elaborate inheritance system for managing access. This should serve as a reminder for any admin.

**Review by Kharonus — CHANGES_REQUESTED** [review]
WDYT about those change requests?

**Comment by MayaBerd** [comment]
> WDYT about those change requests?

@Kharonus very helpful, thank you! I've made the changes.

**Pull request merged**